### PR TITLE
Created Billing endpoint

### PR DESCRIPTION
**Pull request by mikeopensauce**
I don't know if you deliberately haven't included the billing endpoint or you just haven't got round to it but I have added it here as I needed to fetch transactions for my customers. Feel free to add it if you want. It's just a clone of other work you've carried out.
Thanks
Mike

### CODE REVIEW

File: src/APIs/Billing.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    /**
+     * Gets the transaction details for the specified Customer Id.
+     *
+     * @param $customerId
+     *
+     * @return mixed|SimpleXMLElement
+     */
+    public function customerTransactions($customerId)
+    {
+        return $this->get('customer-transactions', ['customer-id' => $customerId, 'transaction-ids' => 99210620]);
```
Comment (rytisder) [inline src/APIs/Billing.php]: I think you left some hardcoded values here: 
```'transaction-ids' => 99210620```

File: src/APIs/Billing.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+     * @var string
+     */
+    protected $api = 'billing';
+
+    /**
+     * Gets the transaction details for the specified Customer Id.
+     *
+     * @param $customerId
+     *
+     * @return mixed|SimpleXMLElement
```
Comment (rytisder) [inline src/APIs/Billing.php]: Fix PHPDoc, it's gonna return array here

File: src/APIs/Billing.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+     * @param array  $transactionId
+     * @param string $transactionDescription
+     * @param string $balanceType
+     * @param int    $amtRangeStart
+     * @param int    $amtRangeEnd
+     * @param string $transactionDateStart
+     * @param string $transactionDateEnd
+     * @param string $orderBy
+     *
+     * @return mixed
```
Comment (rytisder) [inline src/APIs/Billing.php]: Fix PHPDoc - array will be returned

**Review by rytisder — CHANGES_REQUESTED** [review]

**Comment by mikeopensauce** [comment]
Oops!

Mike Cox
Managing Director
The Rosetta Group Ltd

Email: mike@rosetta.group
Website: www.rosetta.group<http://www.rosetta.group>
Phone: 01983 229 229

￼

Have a look at our new cloud phone system at www.rosettatelecom.com<http://www.rosettatelecom.com>
See our payment options and pay online by clicking here Pay online

________________________________

File: src/APIs/Billing.php (hunk 1 above)
Comment (mikeopensauce) [reply src/APIs/Billing.php]: Always check twice :)

File: src/APIs/Billing.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+    /**
+     * @var string
+     */
+    protected $api = 'billing';
+
+    /**
+     * Gets the transaction details for the specified Customer Id.
+     *
+     * @param $customerId
```
Comment (rytisder) [inline src/APIs/Billing.php]:
```suggestion
     * @param int $customerId
```

File: src/APIs/Billing.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+     *
+     * @return array
+     */
+    public function customerTransactions($customerId)
+    {
+        return $this->get('customer-transactions', ['customer-id' => $customerId]);
+    }
+
+        /**
```
Comment (rytisder) [inline src/APIs/Billing.php]:
```suggestion
   /**
```

**Comment by mikeopensauce** [comment]
Thanks for the suggested changes. Always happy to learn!
Mike

**Comment by rytisder** [comment]
@mikeopensauce, happy to help! 🚀 🥳

**Comment by rytisder** [comment]
@amirkhiz, It would be great if you would merge this PR :) Mike, probably, needs to have this feature asap :)

**Comment by rytisder** [comment]
I could do it, but I need write access for this repo. `(Only those with write access to this repository can merge pull requests)` Would you mind giving to me write access?

**Comment by amirkhiz** [comment]
Hey,
Sorry I missed a lot of conversations :).
Thank you @mikeopensauce  for adding Billing part of this APIs it is a great idea.
Many thanks @rytisder for your reviews. OK, Sure I will give you write access to merge PRs. 👍

**Comment by amirkhiz** [comment]
@rytisder I just sent you an invitation.

**Comment by amirkhiz** [comment]
@mikeopensauce v1.5.2 is now available with Billing APIs. You can update to this version.

**Pull request merged**